src/runner/nunit.runner.Droid/project.json

```diff
@@ -1,8 +1,8 @@
 ﻿{
   "dependencies": {
-    "NUnit": "[3.8.1]",
+    "NUnit": "3.9.0",
```

Comment:
This should still be fixed to a particular version - I assume removing it was accidental? (This has happened in many of the project.json's)

Reply:
Actually, where these auto-removed when you upgraded? I wonder if switching to new-csproj would help - `dotnet migrate` has been totally pain-free on the things I've tried it on so far...

Reply:
Yes, it was when I upgraded in the UI. I would like to switch these to the new project format and maybe even switch the three platform specific projects to .NET Standard and just distribute that.